public/js/main.js

```diff
@@ -0,0 +1,25 @@
+
+$("#newGame").click(function(){
+	$("td").removeClass()});
+
+$('.cell').click(function(event){
+	var selectedCell = event.target
+	$.post('/set_cell', { cell: selectedCell.id }).done(function(data){
```

Comment:
It would be great to do some error handling after your `.done` method. See https://api.jquery.com/jquery.post/... in this case, in case of an error, it would be good to inform your user that there's a problem beyond the game just not working.